app/views/maps/map.html.haml

```diff
@@ -115,7 +115,7 @@
     document.getElementById('address').value = "";
     document.getElementById('by_city_name').value = "";
     document.getElementById('by_state_name').value = "";
-    document.getElementById('#by_city_no_state').value = "";
```

Comment:
I assume the extra `#` here was unintentional

Reply:
Yeah, must have been a mistake. Not sure why I didn't notice errors, though.